provisioning_templates/finish/preseed_default_finish.erb

```diff
@@ -15,10 +15,12 @@ oses:
 %>
 
 <% subnet = @host.subnet -%>
-<% if subnet.respond_to?(:dhcp_boot_mode?) -%>
```

Comment:
@ares do you happen to know why subnet would not respond to this? Is this just to prevent `nil` error?

Reply:
subnet can indeed be nil, but I wonder if we also introduced this a long time ago to be compatible with Foreman versions that didn't have the boot mode method yet.

Reply:
I thought so, we are good then.